### DISCUSSION TITLE
Which tool did you use to generate the main image with the screenshots?

### DISCUSSION THREAD

**Discussion by mdmintz** [Textualize/textual#96] [General]
Hello, I'm just wondering which tool you used to create the main image with the angled screenshots and shadow/glow effect on the edges. That looks quite useful. Thanks! And excellent work on this!

![](https://github.com/willmcgugan/textual/blob/main/imgs/textual.png?raw=true)

**Comment by willmcgugan** [comment]
The drop shadow is just what you get when you screenshot a window on MacOS. The perspective effect is Photoshop.

**Reply by mdmintz** [reply]
Thanks! :)
